pynamodb/attributes.py

```diff
@@ -585,8 +585,6 @@ def __init__(self, hash_key=False, range_key=False, null=None, default=None, att
                                             default=default,
                                             attr_name=attr_name)
         if of:
-            if not issubclass(of, MapAttribute):
```

Comment:
mmm this isn't right. `of` is not required.
the doc is incorrect.

[here](https://github.com/pynamodb/PynamoDB/blob/0b3349d5d2b1b646c2889524516624eb51a01249/pynamodb/tests/test_model.py#L3001) is a raw (untyped) list attribute example.

I updated the docs [here](https://github.com/pynamodb/PynamoDB/pull/282)